### ISSUE TITLE
WebRTCModule from 'reactive-native' is NULL

### ISSUE THREAD

**Issue by zhoub** [oney/react-native-webrtc#603]
#### Expected behavior

WebRTCModule should not be the NULL.

#### Observerd behavior

Return NULL.

#### Steps to reproduce the problem

Install the latest one
```
npm install react-native-webrtc
```

Add the following code at the beginning of App.js, the console always prints that the moduels are null objects.
```
import {DeviceEventEmitter, NativeModules} from 'react-native';
const {WebRTCModule} = NativeModules;
```
```
console.log('@@@ ', NativeModules, WebRTCModule, ' ###')
```

```
2019-03-17 17:44:31.389381+0900 RtcApp[1614:47668] '@@@ ', {}, null, ' ###'
```

#### Platform information

iOS

* **React Native version**: 0.59.1
* **Plugin version**: 1.69.0
* **OS**: iOS 
* **OS version**: 12.1

**Comment by saghul** [comment]
Have you lined the module?

**Comment by flyinghawker** [comment]
@saghul @zhoub 
The problem also happens to me.
I tried both in simulator and device and have similar problem.
When try with video option:

> The issue could be traced back to Line 23, MediaDevices.js


When audio:

> Line 116, Permissions.js


Both result from WebRTCModule null issue.
I did both of manual link and also react-native link react-native-webrtc.

> Also repeat work in Appendix A.


I also tried demo of oney(outdated) and latest one from link below
[https://github.com/DimitrisTzimikas/rnWebRTC/blob/master/src/App.js](url)



iOS: 12.2
Xcode: 10.2.1
Simulator: iPhone X
Device: iPhone X Max
Package version: 1.69.1
